src/packages/end-to-end/scripts/import-sqlite.ts

```diff
@@ -24,34 +15,32 @@ async function copyFile(source: string, destination: string) {
 
 async function main() {
 	try {
+		execSync('pwd', { stdio: 'inherit' });
 		await removeDirectory('./app');
 
 		// Create a new instance of the app
-		const init = execSync(
-			'node ../cli/bin init --name=app --backend=sqlite --useVersion="local"'
-		).toString();
-		console.log(init);
+		execSync(
```

Comment:
Should we make these `await exec(` from `child_process/promises` calls because we know about the `execSync` thing where it keeps open handles around?